Sources/Kommunicate/Classes/KMConversationViewController.swift

```diff
@@ -135,6 +135,8 @@ open class KMConversationViewController: ALKConversationViewController {
         guard let channelId = viewModel.channelKey else { return }
         sendConversationOpenNotification(channelId: String(describing: channelId))
         setupConversationClosedView()
+        delayInterval = KMAppUserDefaultHandler.shared.botMessageDelayInterval/1000
+        UserDefaults.standard.set((delayInterval), forKey: "botDelayInterval")
```

Comment:
Why can't we just use the above key-value instead of saving it again?

Reply:
removed..used the same key-value pair everywhere